Pull 191

JenniferMack opened the pull request:
Here’s a PR that adds the home button on author/post/page/tag even when there’s a nav panel. I think it looks a bit nicer to have something in the top left if the blog logo isn’t set. It’s kind of an edge case, see what you think.

keiga-san commented:
Yes, I think it's a great idea. Thanks.

I forgot about the logo as button. But I also think we need to hide the Home button in situations where logo is set and the Navigation is not. Can you add it? 

I think it should looks something like this:

	{{#if @blog.navigation}}
	<a class="menu-button icon-menu" href="#"><span class="word">Menu</span></a>
	{{#unless @blog.logo}}
	<a class="back-button icon-arrow-left" href="{{@blog.url}}/#content">Home</a>
	{{/unless}}
	{{else}}
	{{#unless @blog.logo}}
	<a class="back-button icon-arrow-left" href="{{@blog.url}}">Home</a>
	{{/unless}}
	<a class="subscribe-button-standalone icon-feed" href="{{@blog.url}}/rss/">Subscribe</a>
	{{/if}}

JenniferMack commented:
Yeah, that makes sense. On my way to bed, I’ll dig into it once the sun is up.

-- 
Best regards,

Jennifer Mack
www.jennifermack.net


> On Apr 2, 2015, at 1:51 AM, Keiga <notifications@github.com> wrote:
> 
> Yes, I think it's a great idea. Thanks.
> 
> I forgot about the logo as button. But I also think we need to hide the Home button in situations where logo is set and the Navigation is not. Can you add it?
> 
> I think it should looks something like this:
> 
> {{#if @blog.navigation}}
> <a class="menu-button icon-menu" href="#"><span class="word">Menu</span></a>
> {{#unless @blog.logo}}
> <a class="back-button icon-arrow-left" href="{{@blog.url}}/#content">Home</a>
> {{/unless}}
> {{else}}
> {{#unless @blog.logo}}
> <a class="back-button icon-arrow-left" href="{{@blog.url}}">Home</a>
> {{/unless}}
> <a class="subscribe-button-standalone icon-feed" href="{{@blog.url}}/rss/">Subscribe</a>
> {{/if}}
> 
> —
> Reply to this email directly or view it on GitHub.
>

JenniferMack commented:
Updated. I changed the logic a bit. The corners are independent of each other now. Left corner get blog icon or home button. Right corner get subscribe button or menu. Plus it got rid of a duped line. 

I might have screwed up the whitespace, but it was making my eye twitch like it was. :)

keiga-san commented:
Now things look just great :) Thank you for help.

JenniferMack commented:
You’re welcome, glad to help.
